<!--
  This file is part of drone_GPS.

  Author: Juan Pablo Rodríguez, Pontificia Universidad Javeriana - Bogotá <j_rodriguezg@javeriana.edu.co>
-->
<launch>
	<node name="drone_stateestimation" pkg="tum_ardrone" type="drone_stateestimation">
	</node>
        <node name="GPS" pkg="drone_GPS" type="GPS" output="screen">
        </node>  
	<node name="GPS_test" pkg="drone_GPS" type="GPS_test" output="screen">
        </node>   
	  
</launch>
